Tests/XRPLSwift-restTests/Bip39Mnemonics_Test.swift

```diff
@@ -98,4 +98,12 @@ struct Bip39Mnemonics {
         
         #expect(wallet.classicAddress == expectedAddress)
     }
+    
+    @Test func testSign() async throws {
+        let wallet = Wallet.fromSeed("sEdV6bhEV489Abx3EamsUdk8chZA34i")
+        let paymentTx = Payment(amount: Amount.string("10"), destination: "r3UNMj8BuMmSmR5EYAseWq7RXcNCmzZTk2")
+        
+        var unsignedTx = try paymentTx.toJson()
+        _ = try! wallet.sign(unsignedTx)
```

Comment:
I think it's okay to use expect since the signature won't change.

Reply:
[dafef2d](https://github.com/girin-app/XRPLSwift-rest/pull/4/commits/dafef2d5c2c7b0d228d94628723a5ecc27765565)